packages/shared/src/types/misc.ts

```diff
@@ -0,0 +1,31 @@
+import { ArenaMatchEndInfo, ArenaMatchStartInfo, CombatResult, WowVersion } from '@wowarenalogs/parser';
+
+/** TODO: MUSTFIX move these to somewhere move centralized */
```

Comment:
agree.. seen this same definition in multiple places now

Reply:
yeah all the stuff marked MUSTFIX will get changed before this is ready to merge